Source/ThirdParty/TurboBadger/tb_widgets.cpp

```diff
@@ -359,7 +359,8 @@ void TBWidget::AddChildRelative(TBWidget *child, WIDGET_Z_REL z, TBWidget *refer
 
 void TBWidget::RemoveChild(TBWidget *child, WIDGET_INVOKE_INFO info)
 {
-    assert(child->m_parent);
+    if (!child->m_parent)
+        return;
 
```

Comment:
As it stands I would say it is a side effect of another call, I will need to look into it. I suspect it has something to do with the order in which I close the scene editor and remove the Animation previewer widget...